Sitefinity CLI/PackageManagement/NuGetApiClient.cs

```diff
@@ -135,16 +163,34 @@ private async Task<XDocument> GetPackageXmlDocument(string id, string version, I
             }
 
             HttpResponseMessage response = null;
-            foreach (string source in sources)
+            var v3Sources = sources.Where(x => x.Contains(Constants.ApiV3SourceString));
+            var v2Sources = sources.Except(v3Sources);
+            foreach (string source in v3Sources)
```

Comment:
extract into a method for better readability.

Reply:
the code is refactored